src/EventSubscriber.ts

```diff
@@ -30,7 +27,7 @@ export default class EventSubscriber
   _listen(event: string) {
     if (this._listeners.has(event)) return;
 
-    const listener = data => {
+    const listener = (data: any) => {
```

Comment:
there should be some sort of typing between this and the credentials checker, maybe?